shared/base.js

```diff
@@ -14,7 +14,18 @@ function createContextMenu() {
   });
 }
 
-function enableOrDisable(tabId, changeInfo, tab) {
+function enableOrDisableOnActive(activeInfo) {
+  let activeId = activeInfo.tabId;
```

Comment:
nit: should be const here? since the variable is not reassigned later on

Reply:
yeah, good point.